app/src/index.html

```diff
@@ -69,34 +75,79 @@
-                                <input v-model="todoText" placeholder="What needs to be done" class="form-control" v-on:keyup.enter="addTodo" maxlength=144
-                                    type="text" />
+                                <input v-model="todoText" placeholder="What needs to be done" class="form-control"
+                                    v-on:keyup.enter="addTodo" maxlength=144 type="text" />
                             </div>
                             <ul v-if="todos.length">
                                 <li v-for="el in pendingTodos" class="todos_items">
                                     <input type="checkbox" v-on:click="el.done = !el.done" checked="{{el.done}}">
                                     <label style="font-weight: normal;margin-bottom: 0px;"> {{ el.text }} </label>
-                                    <span class="delete-btn pull-right" v-on:click="el.deleted = true" title="Delete">x</span>
+                                    <span class="delete-btn pull-right" v-on:click="el.deleted = true"
+                                        title="Delete">x</span>
                                 </li>
                                 <div v-if="showAll">
                                     <li v-for="el in doneTodos" class="todos_items done-todos">
-                                        <input type="checkbox" value="{{el.todoId}}" v-on:click="el.done = !el.done" checked="{{el.done}}">
-                                        <label class="done-true" style="font-weight: normal;margin-bottom: 0px;"> {{ el.text }} </label>
-                                        <span class="delete-btn pull-right" v-on:click="el.deleted = true" title="Delete">x</span>
+                                        <input type="checkbox" value="{{el.todoId}}" v-on:click="el.done = !el.done"
+                                            checked="{{el.done}}">
+                                        <label class="done-true" style="font-weight: normal;margin-bottom: 0px;">
+                                            {{ el.text }} </label>
+                                        <span class="delete-btn pull-right" v-on:click="el.deleted = true"
+                                            title="Delete">x</span>
                                     </li>
                                 </div>
                                 <div @click="showAll = !showAll" v-show="doneTodos.length" class="show-more-toggle">
-                                    <span >{{showAll ? 'Hide' : 'Show'}} completed items {{showAll ? '[-]' : '[+]'}}</span>
+                                    <span>{{showAll ? 'Hide' : 'Show'}} completed items
+                                        {{showAll ? '[-]' : '[+]'}}</span>
                                 </div>
                             </ul>
                         </div>
                     </div>
                 </div>
+                <div class="row">
```

Comment:
@alok722, Can you please check https://developer.chrome.com/extensions/options once to provide customizations in the Chrome Extension standard. This way even more options can be added on ad-hoc basis without the need to tamper the UI while doing so.

Note: https://developer.chrome.com/extensions/storage#property-sync will be needed instead of localStorage to utilize the feature.